File: JAVA/bumjinlee/Project/src/Project0107/Homework01.java
package Project0107;

class OutPut1 {
    final int Hold = 10;

    public void Num() {
        for (int i = 1; i<=Hold; i++){
            System.out.println("1부터 10까지 출력 = " + i);
        }
    }
}
public class Homework01{
    //1) 1 ~ 10 까지 출력하는 클래스내의 매서드를 만들어보자!
    public static void main(String[] args) {
        OutPut1 num = new OutPut1();
        num.Num();
    }
}

File: JAVASCRIPT/junghyunlee/js_test/src/homeWork-210201/HomeWork01.js
const HomeWork01 = () => {
    //  1, 1, 1, 3, 5, 9, 17, 31, 57, 105, 193, ......
    // 첫째, 둘째, 셋째 더해서 넷째 출력.

    // 내가 원하는 항의 값을 출력
    // 숫자 입력
    let oneNum = 1

    function inputNumber(num) {

        let firstNum = oneNum;
        let SecondNum = oneNum;
        let ThirdNum = oneNum;
        let totalNum

        if(num == 1 || num == 2 || num ==3){
            console.log("%d 항의 출력 값은 = %d", num, 1)
        } else {
        for (var i = oneNum; i <= num - 3 ; i++) {

            totalNum = firstNum + SecondNum + ThirdNum

            firstNum = SecondNum
            SecondNum = ThirdNum
            ThirdNum = totalNum
            }
            console.log("%d 항의 출력 값은 = %d", num, totalNum)
        }

        return num
    }

    let a = inputNumber(10)
    let b = inputNumber(1)
    let c = inputNumber(5)

    return (
        <div className="HomeWork01">
            <p>
                Home Work01
            </p>
        </div>
    )
}

export default HomeWork01